src/contriblevels.nut

```diff
@@ -18,12 +18,7 @@
 						name = function() { return contribName }
 						func = function() {
 							lastLevelsCounted = {"contribFolder":null, "completed":null, "total":null, "percentage":null}
-							game=clone(gameDefault)
-							game.completed.clear()
-							game.allCoins.clear()
-							game.allEnemies.clear()
-							game.allSecrets.clear()
-							game.bestTime.clear()
```

Comment:
5 of the arrays/objects are explicitly `cleared`; however, the `game` object actually has several other arrays/objects (notably, `colorswitch`, `characters` , `secretOrbs`, `friends`).

We could invoke, e.g., `game.colorswitch = [false, false, false, false, ...]; game.characters = ...`, but I figured it would be simpler to just have a function that re-creates the actual new game object.